catalyst/contrib/dl/callbacks/periodical_loader.py

```diff
@@ -0,0 +1,121 @@
+from typing import Mapping
+from collections import OrderedDict
+import copy
+
+from torch.utils.data import DataLoader
+
+from catalyst.core import Callback, CallbackOrder, State
+
+
+class PeriodicLoaderRunnerCallback(Callback):
```

Comment:
is it possible to rename it to something without `Runner` (it could be misleading for users)
`PeriodicLoaderRunCallback` ?  or just `PeriodicLoaderCallback`? what do you think about it?

Reply:
Sure, I'll use `PeriodicLoaderCallback`